gateway-ha/src/main/java/com/lyft/data/gateway/ha/resource/PrestoResource.java

```diff
@@ -44,55 +48,175 @@ public Response createResourceGroup(String jsonPayload) {
+  }
+
+  @Path("/selector/update")
+  @POST
+  public Response updateSelector(String jsonPayload) {
+    try {
+      SelectorsDetail selector = OBJECT_MAPPER.readValue(jsonPayload, SelectorsDetail.class);
+      SelectorsDetail updatedSelector = this.resourceGroupsManager.updateSelector(selector);
+      return Response.ok(updatedSelector).build();
+    } catch (IOException e) {
+      log.error(e.getMessage(), e);
+      throw new WebApplicationException(e);
+    }
+  }
+
+  @Path("/selector/delete/{resourceGroupId}")
+  @POST
+  public Response deleteSelector(@PathParam("resourceGroupId") String resourceGroupIdStr) {
+    if (Strings.isNullOrEmpty(resourceGroupIdStr)) { // if query not specified, return all
+      throw new WebApplicationException("EntryType can not be null");
+    }
+    long resourceGroupId = Long.parseLong(resourceGroupIdStr);
+    resourceGroupsManager.deleteSelector(resourceGroupId);
+    return Response.ok().build();
+  }
+
+  @POST
+  @Path("/globalproperty/create")
+  public Response createGlobalProperty(String jsonPayload) {
+    try {
+      GlobalPropertiesDetail globalProperty =
+          OBJECT_MAPPER.readValue(jsonPayload, ResourceGroupsManager.GlobalPropertiesDetail.class);
+      GlobalPropertiesDetail newGlobalProperty =
+          this.resourceGroupsManager.createGlobalProperty(globalProperty);
+      return Response.ok(newGlobalProperty).build();
+    } catch (IOException e) {
+      log.error(e.getMessage(), e);
+      throw new WebApplicationException(e);
+    }
+  }
+
+  @GET
+  @Path("/globalproperty/read")
+  public Response readAllGlobalProperties() {
+    return Response.ok(this.resourceGroupsManager.readAllGlobalProperties()).build();
+  }
+
+  @GET
+  @Path("/globalproperty/read/{name}")
+  public Response readGlobalProperty(@PathParam("name") String name) {
+    if (Strings.isNullOrEmpty(name)) {
+      return Response.ok(this.resourceGroupsManager.readAllGlobalProperties()).build();
+    }
+    List<GlobalPropertiesDetail> globalProperty =
+        this.resourceGroupsManager.readGlobalProperty(name);
+    return Response.ok(globalProperty).build();
+  }
+
+  @Path("/globalproperty/update")
+  @POST
+  public Response updateGlobalProperty(String jsonPayload) {
+    try {
+      GlobalPropertiesDetail globalProperty =
+          OBJECT_MAPPER.readValue(jsonPayload, ResourceGroupsManager.GlobalPropertiesDetail.class);
+      GlobalPropertiesDetail updatedGlobalProperty =
+          this.resourceGroupsManager.updateGlobalProperty(globalProperty);
+      return Response.ok(updatedGlobalProperty).build();
+    } catch (IOException e) {
+      log.error(e.getMessage(), e);
+      throw new WebApplicationException(e);
+    }
+  }
+
+  @Path("/globalproperty/delete/{name}")
+  @POST
+  public Response deleteGlobalProperty(@PathParam("name") String name) {
+    resourceGroupsManager.deleteGlobalProperty(name);
+    return Response.ok().build();
+  }
+
   //  @POST
-  //  @Path("/selector/create")
```

Comment:
can we use /* */ instead of slashes to comment out the code?

Also, please add a comment explaining why we commented the code out.